index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:200,300,300i,400,700&display=swap" rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <link rel="stylesheet" href="stylesheets/animate_name.css">
+
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <!-- Header -->
+    <header class="header">
+        <div class="container">
+            <nav class="nav" id="nav">
+                <a class="nav__link" href="index.html">home</a>
+                <a class="nav__link" href="pages/portfolio.html">projects</a>
+                <a class="nav__link" href="pages/about.html">about me</a>
+                <a class="nav__link" href="pages/blog.html">blog</a>
+                <a class="nav__link" href="pages/contact.html">contact</a>
+            </nav>
+        </div>
+    </header>
+
+    <!-- Intro -->
+    <section class="intro">
+        <div class="container">
+            <div class="intro__inner">
+                <div class="intro__content">
```

Comment:
Style: This is very deeply nested.  Generally it's a good idea to do a pass after you finish coding to see if you can eliminate some unnecessary containers.